doc/release-notes/5.8-release-notes.md

```diff
@@ -0,0 +1,120 @@
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [5.8 Milestone](https://github.com/IQSS/dataverse/milestone/99?closed=1) in Github.
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Community Google Group](https://groups.google.com/forum/#!forum/dataverse-community) or email support@dataverse.org.
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](https://guides.dataverse.org/en/5.8/installation/). Please also contact us to get added to the [Dataverse Project Map](https://guides.dataverse.org/en/5.8/installation/config.html#putting-your-dataverse-installation-on-the-map-at-dataverse-org) if you have not done so already.
+
+## Upgrade Instructions
+
+0\. These instructions assume that you've already successfully upgraded from Dataverse Software 4.x to Dataverse Software 5 following the instructions in the [Dataverse Software 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0). After upgrading from the 4.x series to 5.0, you should progress through the other 5.x releases before attempting the upgrade to 5.8.
+
+If you are running Payara as a non-root user (and you should be!), **remember not to execute the commands below as root**. Use `sudo` to change to that user first. For example, `sudo -i -u dataverse` if `dataverse` is your dedicated application user.  
+
+In the following commands we assume that Payara 5 is installed in `/usr/local/payara5`. If not, adjust as needed.
+
+`export PAYARA=/usr/local/payara5`
+
+(or `setenv PAYARA /usr/local/payara5` if you are using a `csh`-like shell)
+
+1\. Undeploy the previous version.
+
+- `$PAYARA/bin/asadmin list-applications`
+- `$PAYARA/bin/asadmin undeploy dataverse<-version>`
+
+2\. Stop Payara and remove the generated directory
+
+- `service payara stop`
+- `rm -rf $PAYARA/glassfish/domains/domain1/generated`
+
+3\. Start Payara
+
+- `service payara start`
+  
+4\. Deploy this version.
+
+- `$PAYARA/bin/asadmin deploy dataverse-5.8.war`
+
+5\. Restart payara
+
+- `service payara stop`
+- `service payara start`
+
+## Additional Release Steps
+
+1\. Replace Solr schema.xml. See specific instructions below for those installations without custom metadata blocks (1a) and those with (1b).
+
+1a\.
+
+For installations without Custom Metadata Blocks:
+
+- stop Solr instance (usually service solr stop, depending on Solr installation/OS, see the [Installation Guide](https://guides.dataverse.org/en/5.8/installation/prerequisites.html#solr-init-script)
+
+- replace schema.xml with the file included in this release
+
+cp /tmp/dvinstall/schema.xml /usr/local/solr/solr-8.8.1/server/solr/collection1/conf
+
+- start Solr instance (usually service solr start, depending on solr/OS)
+
+2\. Run a full reindex, following the steps in the [Admin Guide](http://guides.dataverse.org/en/5.8/admin/solr-search-index.html).
```

Comment:
This line seems out of order (I'm assuming it was meant to go after 1b.).
But I don't think it's needed - I don't think we need to reindex anything (because nobody has any embargoed files just yet - no?)